Zend/zend_compile.h

```diff
@@ -407,9 +407,11 @@ typedef struct _zend_property_info {
 
 typedef struct _zend_class_constant {
 	zval value; /* flags are stored in u2 */
+	zend_string *name;
```

Comment:
No, we don't really need it indeed. Thanks for catching this!

Reply:
With my latest push, it's re-added (I removed it previously) because the field is now used in multiple places. But nevertheless, I think it's useful anyway to have it so that 1.) opcache unserializes the string as an interned string 2.) we have consistency with global constants